QuizWhois.Domain/Migrations/20211028152918_Hint.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace QuizWhois.Domain.Migrations
+{
+    public partial class Hint : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "RatingNumber",
+                table: "QuestionRating",
+                newName: "Value");
+
+            migrationBuilder.AddColumn<long>(
+                name: "QuizId",
+                table: "Question",
+                type: "bigint",
+                nullable: true);
+
+            migrationBuilder.CreateTable(
+                name: "Hint",
+                columns: table => new
+                {
+                    Id = table.Column<long>(type: "bigint", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    Text = table.Column<string>(type: "nvarchar(255)", maxLength: 255, nullable: false),
+                    QuestionId = table.Column<long>(type: "bigint", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Hint", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Hint_Question_QuestionId",
+                        column: x => x.QuestionId,
+                        principalTable: "Question",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateTable(
```

Comment:
It's not a Question table, it's a Quiz one, guess it is the first change on master since my merge request, in which i discarded all db interactions. So there's nothing wrong here